R/assemble.R

```diff
@@ -241,3 +252,55 @@ de_duplicate_names <- function(x) {
   )
   x
 }
+
+
+# does a data frame contain duplicate rows
+# @param x a data.frame
+# @return logical, TRUE if there are any duplicates in the data.frame
+contains_duplicates = function(x) {
+  sum(find_duplicates(x)) > 0
+}
+
+# @param x a data.frame
+# @return a logical vector of the duplicated rows
+find_duplicates = function(x) {
+  duplicated(dplyr::select_if(x, is.atomic))
+}
+
+# @param x a data.frame
+duplicate_drop = function(x) {
+  ix_dupes = find_duplicates(x)
+  if (sum(ix_dupes) > 0) {
+    warning(
+      "There are duplicate responses, duplicates are dropped in
+      the results. Set fix_duplicates = 'keep' to retain them."
```

Comment:
```suggestion
      paste0("There are ", ix_dupes, " duplicate responses, duplicates are dropped in the results. Set fix_duplicates = 'keep' to retain them.")
```
Not necessary, but would be a good addition to alert the user about the # of duplicate responses that are returned.

Reply:
Your suggested change didn't quite give the message you intended, have pushed a change that does